ragavee/CommandLine.java

```diff
@@ -0,0 +1,11 @@
+class CommandLine{
+    public static void main(String args[]){
+        if(args.length>0){
```

Comment:
Great! You have learned command line arguments,,, but can u implement in any prob. It should atleast have some coding complexity